components/vocabularies/chebi/api/src/test/java/org/phenotips/vocabulary/internal/solr/ChEBIOntologyTest.java

```diff
@@ -88,7 +88,6 @@ public void testChEBIOntologyReindex()
         Mockito.verify(this.server, Mockito.atLeast(1)).commit();
         Mockito.verify(this.server, Mockito.atLeast(1)).add(Matchers.anyCollectionOf(SolrInputDocument.class));
         Mockito.verify(this.cache, Mockito.atLeast(1)).removeAll();
-        Mockito.verifyNoMoreInteractions(this.cache, this.server);
```

Comment:
why this change btw?

Reply:
because of the interaction with SolrClient to read single-valued schema fields.

Reply:
I think there is a way to say "exactly one interaction"